specification/marketplacecatalog/data-plane/Reviews/preview/2023-01-01-preview/reviews.json

```diff
@@ -0,0 +1,405 @@
+{
+  "x-generator": "NSwag v13.18.2.0 (NJsonSchema v10.8.0.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Rating And Reviews API",
+    "description": "Microsoft Marketplace Rating And Reviews API",
+    "version": "2023-01-01-preview"
+  },
+  "host": "catalogapi.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/products/{productId}/reviews": {
+      "get": {
+        "tags": [
+          "Reviews Public API"
+        ],
+        "summary": "Get reviews by unique product id",
+        "operationId": "Reviews_GetReviewsByOfferId",
+        "x-ms-examples": {
+          "Facets_Get": {
+            "$ref": "./examples/GetProductReviews.json"
+          }
+        },
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "type": "string",
+            "name": "productId",
+            "in": "path",
+            "required": true,
+            "description": "Unique Product Id",
+            "x-nullable": true
```

Comment:
You should avoid the use of `x-nullable: true` anywhere in your API definition. Properties or parameters with no value should simply be omitted from the payload.

Reply:
Done